### PR TITLE
Remove use of travis_fold

### PR DESCRIPTION
**Pull request by kevmoo**
Fixes https://github.com/dart-lang/site-www/issues/2917

### CODE REVIEW

**Comment by kevmoo** [comment]
I know there are no tests here, but I tried this out w/ `site-www` and it worked great!

File: tool/install.sh [hunk 1]
```diff
@@ -15,13 +15,9 @@ else
...
-travis_fold start install.npm_install
-  (set -x; npm install)
-travis_fold end install.npm_install
+(set -x; npm install)
 
 if [[ -z "$CI" || "$TASK" == *build* ]]; then
-  travis_fold start install.bundle
     bundle --version
     (set -x; bundle install)
-  travis_fold end install.bundle
```
Comment (chalin) [inline tool/install.sh:26]: Might it be worth replacing the `travis_fold` calls by `echo` statements? This is what was done in https://github.com/dart-lang/site-www/pull/2921? For example:

>

**Review by chalin — APPROVED** [review]
LGTM

**Review by johnpryan — APPROVED** [review]
LGTM - should we update https://github.com/dart-lang/site-shared too?

**Comment by kevmoo** [comment]
@chalin – I pushed an update w/ your suggestion

**Pull request merged**